python/tvm/micro/build.py

```diff
@@ -123,9 +123,15 @@ class AutoTvmModuleLoader:
     """
 
     def __init__(
-        self, template_project_dir: Union[pathlib.Path, str], project_options: dict = None
+        self,
+        template_project_dir: Union[pathlib.Path, str],
+        project_options: dict = None,
+        build_dir: str = None,
```

Comment:
Can we give `build_dir` a path-like object type instead of just `str`? i.e. `Union[os.PathLike, str]`?

Reply:
done